react/features/filmstrip/components/web/AudioMutedIndicator.js

```diff
@@ -34,6 +34,7 @@ class AudioMutedIndicator extends Component<Props> {
                 className = 'audioMuted toolbar-icon'
                 icon = { IconMicDisabled }
                 iconId = 'mic-disabled'
+                iconSize = { 13 }
```

Comment:
Do you happen to know why this is hardcoded and why it worked okay before without it?

Reply:
If I remember correctly: This was defined as the default value for the prop. I wanted to remove it in order to allow passing of `undefined`. That's why I had to go and add it manually.

Reply:
If I remember correctly: This was defined as the default value for the prop. I wanted to remove it in order to allow passing of `undefined`. That's why I had to go and add it manually.